manifests/init.pp

```diff
@@ -83,7 +83,7 @@
       }
     }
     ubuntu: {
-      if $::operatingsystemmajrelease != '14.04' {
+      if $::operatingsystemmajrelease < '16.04' {
```

Comment:
After fixing the docker_cmd, this is what's causing 14.04 Beaker tests to fail (so good)